scipy/io/matlab/tests/test_mio.py

```diff
@@ -1290,6 +1290,12 @@ def test_opaque():
     assert isinstance(data['parabola'].item()[3].item()[3], MatlabOpaque)
 
 
+def test_opaque_simplify():
+    """Test that we can read a MatlabOpaque object when simplify_cells=True."""
+    data = loadmat(pjoin(test_data_path, 'parabola.mat'), simplify_cells=True)
+    assert isinstance(data['parabola'], MatlabFunction)
```

Comment:
This test does indeed fail before/pass after the patch. I just wanted to try checking with @landoskape from the original issue, since this PR effectively adopts their patch, but they had indicated that using the patch didn't quite work for them:

> One solution to prevent an error is to include (elem.ndim>0) in the _has_struct(elem) function, however, I tried this and can't run the anonymous function after loading into python.

Perhaps that's resolved now, but wanted to double check with them.

Reply:
>  but they had indicated that using the patch didn't quite work for them

My understanding is that they somehow expected the function object to be transformed into a Python object that's `callable()`.

Reply:
Nice! Thanks for checking. @LemonBoy it would obviously be impressive (read:unfeasible) for a matlab anonymous function to be callable when loaded in python. I'm sure that's possible, but my main issue was that the presence of an anonymous function generated an error when simplify_cells=True. However, adding the additional boolean statement (elem.ndim>0) will prevent that error, which is great.